Expensify/App
# Sync workspace member across devices after removing member(s)

### Details

Sync workspace member across device after member(s) removed.

### Fixed Issues

$ https://github.com/Expensify/App/issues/7095

### Tests



1. Open workspace member page with same account on two device
2. Remove member(s) on a device and refresh member page in other device
3. Members of workspace must be same on those two device
4. In android must go back to setting page and then go to member page to update the member list.

### QA Steps



1. Open workspace member page with same account on two device
2. Remove member(s) on a device and refresh member page in other device
3. Members of workspace must be same on those two device
4. In android must go back to setting page and then go to member page to update the member list.

### Tested On


### Screenshots


#### Web


https://user-images.githubusercontent.com/1313124/151729076-e7d437b3-51b1-4dc9-87f5-e146f4b476f2.mp4

#### Mobile Web


https://user-images.githubusercontent.com/1313124/151729110-04065df4-ff0b-499c-85f6-80e8a625601a.mp4

#### Desktop


#### iOS


#### Android


https://user-images.githubusercontent.com/1313124/151729126-27f8df6a-67f1-4e2a-a7dc-6f5cec888f0b.mp4
